content/en/armory-enterprise/Overview/architecture.md

```diff
@@ -9,7 +9,7 @@ aliases:
 
 ## Armory Enterprise architecture
 
-Armory Enterprise is an enterprise version of open source [Spinnaker](https://spinnaker.io/). It is composed of several microservices for resiliency and follows the single-responsibility principle. It allows for faster iteration on each individual component and a more pluggable architecture for custom components.
+Armory Enterprise is an enterprise version of open source [Spinnaker](https://spinnaker.io/). It is composed of several microservices for resiliency and follows the single-responsibility principle. It allows for faster iteration on each individual component and a more pluggable architecture for custom components. See the open source Spinnaker [microservices overview](https://spinnaker.io/docs/reference/architecture/microservices-overview/#system-dependencies) for port mappings and a table of service interdependencies. 
```

Comment:
I might think to add a comma after resiliency but otherwise LGTM

Reply:
Thanks for asking. 
There isn't a subject before "follows" - the sentence uses a compound predicate.  (an independent and dependent clause - single subject with two verb clauses) 
https://developers.google.com/style/commas#commas-separating-independent-from-dependent-clauses

Since the Chicago Manual of Style online requires a subscription, and I'm using my hard copy as a monitor stand, I sometimes use Grammerly (free) to double-check if I fail to accurately remember my grade school English grammar class in which the teacher tortured us by making us diagram sentences. Grammerly isn't _always_ right for the type of content we produce, but it generally is.